diff --git b/opendbc/safety/modes/hyundai_canfd.h a/opendbc/safety/modes/hyundai_canfd.h
index 09717720..d213961d 100644
--- b/opendbc/safety/modes/hyundai_canfd.h
+++ a/opendbc/safety/modes/hyundai_canfd.h
@@ -197,11 +197,11 @@ static bool hyundai_canfd_tx_hook(const CANPacket_t *msg) {
   //   .wheelbase = 2.81,
   // };
 
-  // KIA_SPORTAGE_HEV_2026 values. (most conservative for now) (values can be found on values.py)
+  // KIA_EV9 values. (values can be found on values.py)
   const AngleSteeringParams HYUNDAI_STEERING_PARAMS = {
-      .slip_factor = -0.0006085930193026732,  // calc_slip_factor(VM)
-      .steer_ratio = 13.7,
-      .wheelbase = 2.756,
+      .slip_factor = -0.0005410588125765342,  // calc_slip_factor(VM)
+      .steer_ratio = 16.0,
+      .wheelbase = 3.10,
     };
 
 
