diff --git a/openpilot/cereal/custom.capnp b/openpilot/cereal/custom.capnp
index cfbbbe472aff06d8e4fbc9b75a4cca791c9bbd20..0dce1b4e748043eea6a19b5ea7d8086cf32c8418 100644
--- a/openpilot/cereal/custom.capnp
+++ b/openpilot/cereal/custom.capnp
@@ -363,6 +363,10 @@ struct CarParamsSP @0x80ae746ee2596b11 {
   pcmCruiseSpeed @3 :Bool;
   intelligentCruiseButtonManagementAvailable @4 :Bool;
   enableGasInterceptor @5 :Bool;
+  hkgTuningAngleCustomLimitMaxSpeedKph @6 :Float32;
+  hkgTuningAngleOverrideEffortPercent @7 :Float32;
+  hkgSharedAutonomyEnabled @8 :Bool;
+  hkgSharedAutonomyMode @9 :UInt8;
 
   neuralNetworkLateralControl @2 :NeuralNetworkLateralControl;
 
@@ -447,6 +451,8 @@ struct BackupManagerSP @0xf98d843bfd7004a3 {
 
 struct CarStateSP @0xb86e6369214c01c8 {
   speedLimit @0 :Float32;
+  handsOnWheel @1 :Bool;
+  handsOnWheelValid @2 :Bool;
 }
 
 struct LiveMapDataSP @0xf416ec09499d9d19 {
diff --git a/openpilot/selfdrive/controls/lib/drive_helpers.py b/openpilot/selfdrive/controls/lib/drive_helpers.py
index 35ca45e56e28d6dc2acab17f2b7611fe79689eb6..031b2e7d57ef22d87c3ef5aa5eba1dbbc35ed5bf 100644
--- a/openpilot/selfdrive/controls/lib/drive_helpers.py
+++ b/openpilot/selfdrive/controls/lib/drive_helpers.py
@@ -1,5 +1,6 @@
 import numpy as np
 from openpilot.common.constants import ACCELERATION_DUE_TO_GRAVITY
+from openpilot.common.params import Params
 from openpilot.common.realtime import DT_CTRL, DT_MDL
 
 MIN_SPEED = 1.0
@@ -25,7 +26,7 @@ def smooth_value(val, prev_val, tau, dt=DT_MDL):
   alpha = 1 - np.exp(-dt/tau) if tau > 0 else 1
   return alpha * val + (1 - alpha) * prev_val
 
-def clip_curvature(v_ego, prev_curvature, new_curvature, roll) -> tuple[float, bool]:
+def clip_curvature(v_ego, prev_curvature, new_curvature, roll, params: Params | None = None) -> tuple[float, bool]:
   # This function respects ISO lateral jerk and acceleration limits + a max curvature
   v_ego = max(v_ego, MIN_SPEED)
   max_curvature_rate = MAX_LATERAL_JERK / (v_ego ** 2)  # inexact calculation, check https://github.com/commaai/openpilot/pull/24755
@@ -33,12 +34,22 @@ def clip_curvature(v_ego, prev_curvature, new_curvature, roll) -> tuple[float, b
                           prev_curvature - max_curvature_rate * DT_CTRL,
                           prev_curvature + max_curvature_rate * DT_CTRL)
 
+  if params is None:
+    params = Params()
+  speed_threshold_mps = float(params.get("HkgTuningAngleCustomLimitMaxSpeedKph", return_default=True)) / 3.6
   roll_compensation = roll * ACCELERATION_DUE_TO_GRAVITY
-  max_lat_accel = MAX_LATERAL_ACCEL_NO_ROLL + roll_compensation
-  min_lat_accel = -MAX_LATERAL_ACCEL_NO_ROLL + roll_compensation
+  if v_ego <= speed_threshold_mps:
+    max_lat_accel = 4.2
+    min_lat_accel = -4.2
+  else:
+    max_lat_accel = MAX_LATERAL_ACCEL_NO_ROLL + roll_compensation
+    min_lat_accel = -MAX_LATERAL_ACCEL_NO_ROLL + roll_compensation
   new_curvature, limited_accel = clamp(new_curvature, min_lat_accel / v_ego ** 2, max_lat_accel / v_ego ** 2)
 
-  new_curvature, limited_max_curv = clamp(new_curvature, -MAX_CURVATURE, MAX_CURVATURE)
+  if v_ego <= speed_threshold_mps:
+    limited_max_curv = False
+  else:
+    new_curvature, limited_max_curv = clamp(new_curvature, -MAX_CURVATURE, MAX_CURVATURE)
   return float(new_curvature), limited_accel or limited_max_curv
 
 
diff --git a/openpilot/sunnypilot/selfdrive/car/interfaces.py b/openpilot/sunnypilot/selfdrive/car/interfaces.py
index 4988e863b1e6322e807e7dff248335081ff7d79a..8a98b508f85a268182f8140a81489cfccb996cb8 100644
--- a/openpilot/sunnypilot/selfdrive/car/interfaces.py
+++ b/openpilot/sunnypilot/selfdrive/car/interfaces.py
@@ -118,6 +118,9 @@ def initialize_params(params) -> list[dict[str, Any]]:
   # hyundai
   keys.extend([
     "HyundaiLongitudinalTuning",
+    "HkgSharedAutonomyMode",
+    "HkgTuningAngleCustomLimitMaxSpeedKph",
+    "HkgTuningAngleOverrideEffortPercent",
   ])
 
   # subaru
