diff --git a/openpilot/selfdrive/monitoring/policy.py b/openpilot/selfdrive/monitoring/policy.py
index bd698214acb01d7bb90049cf74a6dc11f25272a5..480afb57aa99c3b294aba8880b0d16763ae974c1 100644
--- a/openpilot/selfdrive/monitoring/policy.py
+++ b/openpilot/selfdrive/monitoring/policy.py
@@ -50,7 +50,7 @@ class DRIVER_MONITOR_SETTINGS:
     self._EYE_THRESHOLD = 0.65
     self._SG_THRESHOLD = 0.9
     self._BLINK_THRESHOLD = 0.865
-    self._PHONE_THRESH = 0.5
+    self._PHONE_THRESH = 0.68
     self._POSE_PITCH_THRESHOLD = 0.3133
     self._POSE_PITCH_THRESHOLD_SLACK = 0.3237
     self._POSE_PITCH_THRESHOLD_STRICT = self._POSE_PITCH_THRESHOLD
