diff --git b/selfdrive/selfdrived/events.py a/selfdrive/selfdrived/events.py
index b03a953a65..7898c972ce 100755
--- b/selfdrive/selfdrived/events.py
+++ a/selfdrive/selfdrived/events.py
@@ -338,9 +338,9 @@ EVENTS: dict[int, dict[str, Alert | AlertCallbackType]] = {
 
   EventName.promptDriverDistracted: {
     ET.PERMANENT: Alert(
-      "Pay Attention",
       "Driver Distracted",
-      AlertStatus.userPrompt, AlertSize.mid,
+      "",
+      AlertStatus.userPrompt, AlertSize.small,
       Priority.MID, VisualAlert.steerRequired, AudibleAlert.promptDistracted, .1),
   },
 
@@ -430,9 +430,9 @@ EVENTS: dict[int, dict[str, Alert | AlertCallbackType]] = {
 
   EventName.steerSaturated: {
     ET.WARNING: Alert(
-      "Take Control",
       "Turn Exceeds Steering Limit",
-      AlertStatus.userPrompt, AlertSize.mid,
+      "",
+      AlertStatus.normal, AlertSize.small,
       Priority.LOW, VisualAlert.steerRequired, AudibleAlert.promptRepeat, 2.),
   },
 
diff --git b/selfdrive/ui/onroad/alert_renderer.py a/selfdrive/ui/onroad/alert_renderer.py
index e8817f24b4..5e5372a0cf 100644
--- b/selfdrive/ui/onroad/alert_renderer.py
+++ a/selfdrive/ui/onroad/alert_renderer.py
@@ -26,7 +26,7 @@ SELFDRIVE_UNRESPONSIVE_TIMEOUT = 10  # Seconds
 
 # Constants
 ALERT_COLORS = {
-  AlertStatus.normal: rl.Color(0, 0, 0, 235),  # Black
+  AlertStatus.normal: rl.Color(0, 0, 0, 128),  # Black with 50% transparency
   AlertStatus.userPrompt: rl.Color(0xFE, 0x8C, 0x34, 235),  # Orange
   AlertStatus.critical: rl.Color(0xC9, 0x22, 0x31, 235),  # Red
 }
diff --git b/selfdrive/ui/qt/onroad/alerts.h a/selfdrive/ui/qt/onroad/alerts.h
index de38d8ffc3..fb6997ff1e 100644
--- b/selfdrive/ui/qt/onroad/alerts.h
+++ a/selfdrive/ui/qt/onroad/alerts.h
@@ -26,7 +26,7 @@ protected:
   };
 
   const QMap<cereal::SelfdriveState::AlertStatus, QColor> alert_colors = {
-    {cereal::SelfdriveState::AlertStatus::NORMAL, QColor(0x15, 0x15, 0x15, 0xf1)},
+    {cereal::SelfdriveState::AlertStatus::NORMAL, QColor(0x15, 0x15, 0x15, 0x80)},
     {cereal::SelfdriveState::AlertStatus::USER_PROMPT, QColor(0xDA, 0x6F, 0x25, 0xf1)},
     {cereal::SelfdriveState::AlertStatus::CRITICAL, QColor(0xC9, 0x22, 0x31, 0xf1)},
   };
