diff --git b/selfdrive/monitoring/helpers.py a/selfdrive/monitoring/helpers.py
index aea81d336f..6a34f5e84e 100644
--- b/selfdrive/monitoring/helpers.py
+++ a/selfdrive/monitoring/helpers.py
@@ -32,9 +32,9 @@ class DRIVER_MONITOR_SETTINGS:
     self._DISTRACTED_PROMPT_TIME_TILL_TERMINAL = 6.
 
     self._FACE_THRESHOLD = 0.7
-    self._EYE_THRESHOLD = 0.5
-    self._BLINK_THRESHOLD = 0.5
-    self._PHONE_THRESH = 0.5
+    self._EYE_THRESHOLD = 0.65
+    self._BLINK_THRESHOLD = 0.865
+    self._PHONE_THRESH = 0.68
 
     self._POSE_PITCH_THRESHOLD = 0.3133
     self._POSE_PITCH_THRESHOLD_SLACK = 0.3237
