diff --git b/selfdrive/controls/lib/drive_helpers.py a/selfdrive/controls/lib/drive_helpers.py
index e28fa3021c..2ada1f36ef 100644
--- b/selfdrive/controls/lib/drive_helpers.py
+++ a/selfdrive/controls/lib/drive_helpers.py
@@ -31,8 +31,10 @@ def clip_curvature(v_ego, prev_curvature, new_curvature, roll):
                           prev_curvature + max_curvature_rate * DT_CTRL)
 
   roll_compensation = roll * ACCELERATION_DUE_TO_GRAVITY
-  max_lat_accel = MAX_LATERAL_ACCEL_NO_ROLL + roll_compensation
-  min_lat_accel = -MAX_LATERAL_ACCEL_NO_ROLL + roll_compensation
+  speed_threshold_mps = 32.0 / 3.6
+  lat_accel_no_roll = 3.6 if v_ego <= speed_threshold_mps else MAX_LATERAL_ACCEL_NO_ROLL
+  max_lat_accel = lat_accel_no_roll + roll_compensation
+  min_lat_accel = -lat_accel_no_roll + roll_compensation
   new_curvature, limited_accel = clamp(new_curvature, min_lat_accel / v_ego ** 2, max_lat_accel / v_ego ** 2)
 
   new_curvature, limited_max_curv = clamp(new_curvature, -MAX_CURVATURE, MAX_CURVATURE)
