diff --git b/common/params_keys.h a/common/params_keys.h
index cad02cb97a..9f3238ee2e 100644
--- b/common/params_keys.h
+++ a/common/params_keys.h
@@ -246,4 +246,5 @@ inline static std::unordered_map<std::string, ParamKeyAttributes> keys = {
     {"HkgTuningAngleCustomLimitMaxSpeedKph", {PERSISTENT | BACKUP, INT, "32"}},
     {"HkgTuningEv9AlertsSpeedKph", {PERSISTENT | BACKUP, INT, "40"}},
     {"HkgTuningOverridingCycles", {PERSISTENT | BACKUP, INT, "17"}},
+    {"UploadsActive", {CLEAR_ON_MANAGER_START, BOOL}},
 };
diff --git b/system/athena/athenad.py a/system/athena/athenad.py
index 42c9cf8a1c..051145d0ad 100755
--- b/system/athena/athenad.py
+++ a/system/athena/athenad.py
@@ -164,6 +164,9 @@ class UploadQueueCache:
       queue: list[UploadItem | None] = list(upload_queue.queue)
       items = [asdict(i) for i in queue if i is not None and (i.id not in cancelled_uploads)]
       Params().put("AthenadUploadQueue", items)
+      queued_items_present = any(item is not None for item in queue)
+      uploads_active = any(item and item.current for item in cur_upload_items.values()) or queued_items_present
+      Params().put_bool_nonblocking("UploadsActive", uploads_active)
     except Exception:
       cloudlog.exception("athena.UploadQueueCache.cache.exception")
 
diff --git b/system/hardware/power_monitoring.py a/system/hardware/power_monitoring.py
index d0a1e623a7..651c60d699 100644
--- b/system/hardware/power_monitoring.py
+++ a/system/hardware/power_monitoring.py
@@ -130,6 +130,10 @@ class PowerMonitoring:
     low_voltage_shutdown = (self.car_voltage_mV < (VBATT_PAUSE_CHARGING * 1e3) and
                             offroad_time > VOLTAGE_SHUTDOWN_MIN_OFFROAD_TIME_S)
     should_shutdown |= self.max_time_offroad_exceeded(offroad_time)
+    if not should_shutdown and offroad_time > 300:
+      uploads_active = self.params.get_bool("UploadsActive")
+      if not uploads_active:
+        should_shutdown = True
     should_shutdown |= low_voltage_shutdown
     should_shutdown |= (self.car_battery_capacity_uWh <= 0)
     should_shutdown &= not ignition
