diff --git b/selfdrive/selfdrived/selfdrived.py a/selfdrive/selfdrived/selfdrived.py
index 33e5357541..b736db7134 100755
--- b/selfdrive/selfdrived/selfdrived.py
+++ a/selfdrive/selfdrived/selfdrived.py
@@ -412,7 +412,10 @@ class SelfdriveD(CruiseHelper):
       turning = abs(desired_lateral_accel) > 1.0
       # TODO: lac.saturated includes speed and other checks, should be pulled out
       if undershooting and turning and lac.saturated:
-        self.events.add(EventName.steerSaturated)
+        # Silence "Turn Exceeds Steering Limit" alert when below custom speed threshold
+        speed_threshold_mps = float(self.params.get("HkgTuningAngleCustomLimitMaxSpeedKph", return_default=True)) / 3.6
+        if CS.vEgo > speed_threshold_mps:
+          self.events.add(EventName.steerSaturated)
 
     # Check for FCW
     stock_long_is_braking = self.enabled and not self.CP.openpilotLongitudinalControl and CS.aEgo < -1.25
