diff --git b/common/params_keys.h a/common/params_keys.h
index 8021d86d31..9f3238ee2e 100644
--- b/common/params_keys.h
+++ a/common/params_keys.h
@@ -243,6 +243,8 @@ inline static std::unordered_map<std::string, ParamKeyAttributes> keys = {
     {"HkgTuningAngleMinTorqueReductionGain", {PERSISTENT | BACKUP, INT, "10"}},
     {"HkgTuningAngleMaxTorqueReductionGain", {PERSISTENT | BACKUP, INT, "100"}},
     {"HkgTuningAngleActiveTorqueReductionGain", {PERSISTENT | BACKUP, INT, "100"}},
+    {"HkgTuningAngleCustomLimitMaxSpeedKph", {PERSISTENT | BACKUP, INT, "32"}},
+    {"HkgTuningEv9AlertsSpeedKph", {PERSISTENT | BACKUP, INT, "40"}},
     {"HkgTuningOverridingCycles", {PERSISTENT | BACKUP, INT, "17"}},
     {"UploadsActive", {CLEAR_ON_MANAGER_START, BOOL}},
 };
diff --git b/selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/angle_tuning_settings.cc a/selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/angle_tuning_settings.cc
index b26762fa12..e541c8c3f2 100644
--- b/selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/angle_tuning_settings.cc
+++ a/selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/angle_tuning_settings.cc
@@ -53,7 +53,22 @@ AngleTunningSettings::AngleTunningSettings(QWidget *parent) : QWidget(parent) {
   });
   second_row->addWidget(hkgAngleMaxTorque);
   list->addItem(second_row);
-  
+
+  auto third_row = new QHBoxLayout();
+  hkgAngleCustomSpeed = new OptionControlSP("HkgTuningAngleCustomLimitMaxSpeedKph", tr("Speed (EV9 Limits)"), tr("Modify EV9 steering limits only below this speed."), "../assets/offroad/icon_blank.png", {10, 40}, 1);
+  connect(hkgAngleCustomSpeed, &OptionControlSP::updateLabels, hkgAngleCustomSpeed, [=]() {
+    this->updateToggles(offroad);
+  });
+  third_row->addWidget(hkgAngleCustomSpeed);
+
+  // EV9 Alerts speed option
+  hkgEv9AlertsSpeed = new OptionControlSP("HkgTuningEv9AlertsSpeedKph", tr("Speed (EV9 Alerts)"), tr("Modify EV9 steering alerts only below this speed."), "../assets/offroad/icon_blank.png", {10, 50}, 1);
+  connect(hkgEv9AlertsSpeed, &OptionControlSP::updateLabels, hkgEv9AlertsSpeed, [=]() {
+    this->updateToggles(offroad);
+  });
+  third_row->addWidget(hkgEv9AlertsSpeed);
+  list->addItem(third_row);
+
   QObject::connect(uiState(), &UIState::offroadTransition, this, &AngleTunningSettings::updateToggles);
 
   main_layout->addWidget(new ScrollViewSP(list, this));
@@ -80,6 +95,28 @@ void AngleTunningSettings::updateToggles(bool _offroad) {
   auto HkgAngleMaxTorqueValue = QString::fromStdString(params.get("HkgTuningAngleMaxTorqueReductionGain")).toInt();
   hkgAngleMaxTorque->setLabel(QString::number(HkgAngleMaxTorqueValue)+"%");
 
+  auto HkgAngleCustomSpeedParam = params.get("HkgTuningAngleCustomLimitMaxSpeedKph");
+  auto HkgAngleCustomSpeedValue = QString::fromStdString(HkgAngleCustomSpeedParam).toInt();
+  if (HkgAngleCustomSpeedParam.empty()) {
+    HkgAngleCustomSpeedValue = 32;
+  } else if (HkgAngleCustomSpeedValue < 10) {
+    HkgAngleCustomSpeedValue = 10;
+  } else if (HkgAngleCustomSpeedValue > 40) {
+    HkgAngleCustomSpeedValue = 40;
+  }
+  hkgAngleCustomSpeed->setLabel(QString::number(HkgAngleCustomSpeedValue)+tr(" km/h"));
+
+  auto HkgEv9AlertsSpeedParam = params.get("HkgTuningEv9AlertsSpeedKph");
+  auto HkgEv9AlertsSpeedValue = QString::fromStdString(HkgEv9AlertsSpeedParam).toInt();
+  if (HkgEv9AlertsSpeedParam.empty()) {
+    HkgEv9AlertsSpeedValue = 40;
+  } else if (HkgEv9AlertsSpeedValue < 10) {
+    HkgEv9AlertsSpeedValue = 10;
+  } else if (HkgEv9AlertsSpeedValue > 50) {
+    HkgEv9AlertsSpeedValue = 50;
+  }
+  hkgEv9AlertsSpeed->setLabel(QString::number(HkgEv9AlertsSpeedValue)+tr(" km/h"));
+
   auto HkgTuningOverridingCyclesValue = QString::fromStdString(params.get("HkgTuningOverridingCycles")).toInt();
   hkgTuningOverridingCycles->setLabel(QString::number(HkgTuningOverridingCyclesValue));
 
diff --git b/selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/angle_tuning_settings.h a/selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/angle_tuning_settings.h
index 7309f600ef..09749ebdec 100644
--- b/selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/angle_tuning_settings.h
+++ a/selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/angle_tuning_settings.h
@@ -35,5 +35,7 @@ private:
   OptionControlSP* hkgAngleMinTorque;
   OptionControlSP* hkgAngleActiveTorque;
   OptionControlSP* hkgAngleMaxTorque;
+  OptionControlSP* hkgAngleCustomSpeed;
+  OptionControlSP* hkgEv9AlertsSpeed;
   OptionControlSP* hkgTuningOverridingCycles;
 };
